Vhar 5316 paikkauskohde valmis kaataa harjan

rckopteri opened the pull request.

rckopteri committed:
Estä mahdollinen takuuajan puuttumisesta johtuva virhe.

Takuuaika tarkistetaan ui:lla, mutta validointi ei ole haitaksi

rckopteri committed:
Lisätään mahdollisuus muokata potin tieosoitetta, jos paikkauskohteen tieosoite jostain syystä muuttuu.

rckopteri committed:
Estä nil valmistumispäivämäärän hyväksyminen pot toteumilla raportoidun paikkauskohteen valmistumislomakkeella

rckopteri committed:
Lisää paikkauskohteesta tehdylle potille päivämäärien pakollisuus

rckopteri committed:
Autetaan käyttäjää hakemalla valmistumispäivämäärä potilta

rckopteri committed:
Poista rikkinäiset lomake/rivi asetukset

rckopteri committed:
Ota käyttöön taustaväri ja aseta valmistumispäivämäärä pakolliseksi

rckopteri committed:
Vahvista pakolliset päivämäärät paikkauskohteen pot lomakkeella

rckopteri committed:
Merge branch 'korjataan-yksikkotesteja' into VHAR-5316-paikkauskohde-valmis-kaataa-harjan

jpluomajoki-solita reviewed (approved):
Näyttäisi korjaavan ongelman. Testauksessa tuli esille muutama omituisuus MH-urakalla, varmistetaan vielä ne ja sitten voi laittaa minun puolesta mergen :+1:

rckopteri committed:
Estetään hoitourakoiden pääsy paikkauskohde sivulle.

Jäljelle jää siis sivu, josta näkevät muiden tekemät paikkauskohteet, mutta omia eivät voi enää tehdä.

rckopteri committed:
Merge branch 'develop' into VHAR-5316-paikkauskohde-valmis-kaataa-harjan
